### ISSUE TITLE
Testing assignments

### ISSUE THREAD

**Issue by jtkulas** [jtkulas/GitHub#3]
This is a test to determine if we can communicate task requests to each other

**Comment by casosorio** [comment]
> This is a test to determine if we can communicate task requests to each other
 
Yes, I think we are. I got email notification saying that this was an issue, and we are able to reply to your comments or issues
